### PR TITLE
feat: allow option to delete orgs

### PR DESCRIPTION
**Pull request by BlankParticle**
## What does this PR do?

Allow org owners to delete orgs
Fixed ENG-115
Fixes ENG-106

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required

- [ ] Read [Contributing Guide](https://github.com/un/inbox/blob/main/CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Tested my code in a local environment
- [ ] Commented on my code in hard-to-understand areas
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the UnInbox Docs if changes were necessary

### CODE REVIEW

**Comment by BlankParticle** [comment]
* **#760** <a href="https://app.graphite.dev/github/pr/un/inbox/760?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @BlankParticle and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web/src/app/[orgShortcode]/settings/org/page.tsx [hunk 1]
```diff
@@ -163,6 +187,85 @@ export default function ProfileComponent() {
...
+      <AlertDialogContent>
+        <AlertDialogTitle>Delete Organization</AlertDialogTitle>
+        <AlertDialogDescription className="flex flex-col gap-2">
+          <span>Are you sure you want to delete this organization?</span>
+          <span className="text-red-10 font-semibold">
+            This action is irreversible. All data associated with this org would
+            be deleted along with all members.
+          </span>
+          <span>
+            If there is any active subscriptions, that would be cancelled.
```
Comment (McPizza0) [inline apps/web/src/app/[orgShortcode]/settings/org/page.tsx:237]: "All active subscriptions will also be cancelled"

File: apps/web/src/app/[orgShortcode]/settings/org/page.tsx [hunk 2]
```diff
@@ -163,6 +187,85 @@ export default function ProfileComponent() {
...
+          </span>
+          <span>
+            If you still wish to delete this organization, please type the
```
Comment (McPizza0) [inline apps/web/src/app/[orgShortcode]/settings/org/page.tsx:240]: "To confirm deletion of ${org.name} - type it's name below."

File: apps/web/src/app/[orgShortcode]/settings/org/page.tsx [hunk 3]
```diff
@@ -163,6 +187,85 @@ export default function ProfileComponent() {
...
+      onOpenChange={() => {
+        if (isPending) return;
+        setOpen(!open);
+      }}>
+      <AlertDialogContent>
+        <AlertDialogTitle>Delete Organization</AlertDialogTitle>
+        <AlertDialogDescription className="flex flex-col gap-2">
+          <span>Are you sure you want to delete this organization?</span>
+          <span className="text-red-10 font-semibold">
+            This action is irreversible. All data associated with this org would
```
Comment (McPizza0) [inline apps/web/src/app/[orgShortcode]/settings/org/page.tsx:233]: use "organization" not org

**Review by McPizza0 — CHANGES_REQUESTED** [review]

**Review by McPizza0 — APPROVED** [review]

**Comment by BlankParticle** [comment]
### Merge activity

* **Aug 31, 9:17 AM EDT**: @BlankParticle merged this pull request with [Graphite](https://app.graphite.dev/github/pr/un/inbox/760).

**Pull request merged**